src/WingetCreateCLI/Commands/NewCommand.cs

```diff
@@ -115,7 +121,56 @@ public override async Task<bool> Execute()
                         return false;
                     }
 
-                    installerUpdateList.Add(new InstallerMetadata { InstallerUrl = installerUrl, PackageFile = packageFile });
+                    if (packageFile.IsZipFile())
+                    {
+                        string extractDirectory = Path.Combine(PackageParser.InstallerDownloadPath, Path.GetFileNameWithoutExtension(packageFile));
+
+                        try
+                        {
+                            ZipFile.ExtractToDirectory(packageFile, extractDirectory, true);

```

Comment:
>extractDirectory [](http://example.com/codeflow?start=68&length=16)

I think remove extractDirectory if exists and create a new directory is more correct than using without checking?